nemo_skills/pipeline/start_server.py

```diff
@@ -51,7 +51,12 @@
     args = parser.parse_args()
 
     cluster_config = get_cluster_config(args.cluster, args.config_dir)
-    check_if_mounted(cluster_config, args.model)
+
+    if args.server_type != "vllm":
+        check_if_mounted(cluster_config, args.model)
+    elif args.server_type == "vllm" and os.path.exists(args.model):
```

Comment:
I think this check is not going to work, since args.model has to be the mounted path, not real path on your file system. So we cannot check if it exists

Reply:
What we can do instead is check that it starts with / since all mounted paths have to be formatted that way

Reply:
I see, good point